src/shared/components/Button/Button.style.ts

```diff
@@ -84,7 +86,34 @@ export const ButtonIconWrapper = styled.span<ButtonIconWrapperProps>`
 
 export const StyledText = styled(Text)<TextProps>`
   /* compensate for line-height being 1 */
-  ${textPaddingStyles};
+  ${textPaddingStyles}
```

Comment:
why the removal of semicolon?

Reply:
Because I was getting a red squiggly line in vscode for it but not sure why

Reply:
On the other hand, this breaks styles for WebStorm:
<img width="489" alt="Screenshot 2021-10-11 at 09 44 11" src="https://user-images.githubusercontent.com/12646744/136751691-39dc7eaf-62f8-4c18-8eac-6a1d1100ea46.png">

If you look at emotion examples, they also add a semicolon after composition, so I think we should use them too: https://emotion.sh/docs/composition

Reply:
yeah I think its a vscode bug I readed it